t/ui/27-plugin_obs_rsync_obs_status.t

```diff
@@ -71,28 +83,95 @@ my %fake_response_by_project = (
     Proj0 => 'invalid XML',
 );
 
+my $auth_header_exact
+  = qq(Signature keyId="dummy-username",algorithm="ssh",)
+  . qq(signature="U1NIU0lHAAAAAQAAADMAAAALc3NoLWVkMjU1MTkAAAAgSKpcECPm8Vjo9UznZS+)
+  . qq(M/QLjmXXmLzoBxkIbZ8Z/oPkAAAAaVXNlIHlvdXIgZGV2ZWxvcGVyIGFjY291bnQAAAAAAAAABn)
+  . qq(NoYTUxMgAAAFMAAAALc3NoLWVkMjU1MTkAAABA8cmvTy1PgpW2XhHWxQ1yw/wPGAfT2M3CGRJ3II)
+  . qq(7uT5Orqn1a0bWlo/lEV0WiqP+pPcQdajQ4a2YGJvpfzT1uBA==",)
+  . qq (headers="(created)",created="1664187470");
+
 note 'Starting fake API server';
-my $server_instance = process sub {
+
+my $server_process = sub {
     my $mock = Mojolicious->new;
     $mock->mode('test');
+
+    my $www_authenticate = qq(Signature realm="Use your developer account",headers="(created)"); # uncoverable statement
+    $mock->routes->get(
+        '/build/ProjWithAuth/_result' => sub {
+            my $c = shift;
+
+            if ($c->req->headers->authorization) {
+                return $c->render(status => 200, text => $fake_response_by_project{Proj1});
+            }
+
+            $c->res->headers->www_authenticate($www_authenticate);
+
+            return $c->render(status => 401, text => 'login');
```

Comment:
```suggestion
            return $c->render(status => 200, text => $fake_response_by_project{Proj1}) if $c->req->headers->authorization;
            $c->res->headers->www_authenticate($www_authenticate);
            return $c->render(status => 401, text => 'login');
```

Reply:
Addressed in the last commit (pushed before noticing the suggestion)